instructor/templates/instructor/assignments.html

```diff
@@ -46,7 +46,11 @@ <h1 class="scb_s_dashboard_sidebar_title">Your Assignments: </h1>
                         </td>
                         <td class='scb_s_dashboard_table_element scb_s_dashboard_table_border'>
                             <a class='scb_ab_s_dashboard_action'
-                               href="{% url "common_assignments_edit" assignment_pk=a.id %}">Edit</a>
+                               href="{% url "common_assignments_edit" assignment_pk=a.id %}">
```

Comment:
When the button changes to `view` I was expecting that clicking it would take me to the student view, not back to editing.